content-src/selectors/selectors.js

```diff
@@ -31,6 +31,21 @@ function getBackgroundRGB(site) {
   return favicon ? DEFAULT_FAVICON_BG_COLOR : getRandomColor(label);
 }
 
+function assignImageAndBackgroundColor(rows) {
```

Comment:
Would you mind actually putting this in its own file, and maybe adding a comment? I know most of the selectors are in the same file right now but it's getting kind of big, and it would be nice if we started refactoring things out